src/browser.ts

```diff
@@ -0,0 +1,5 @@
+import { CREDITS, CreditPricing } from "./index";
+
+// Make it accessible on the window object
+(window as any).BALENA_CREDITS = CREDITS;
```

Comment:
I expect that you would not need this, so maybe we can drop it and polute the global namespace a bit less :)